README.md

```diff
@@ -33,7 +33,7 @@ To further configure your plugin, use the following custom parameters in your `s
 | `logLevel`           | The log level, set to `DEBUG` for extended logging. Defaults to `info`.                                                                                                                                                                                                                                                                                                                           |
 | `enableXrayTracing`  | Set `true` to enable X-Ray tracing on the Lambda functions and API Gateway integrations. Defaults to `false`.                                                                                                                                                                                                                                                                                   |
 | `enableDDTracing`    | Enable Datadog tracing on the Lambda function. Defaults to `true`.                                                                                                                                                                                                                                                                        |
-| `forwarderArn`          | Setting this parameter subscribes the given Datadog forwarder to the Lambda functions’ CloudWatch log groups. Required when `enableDDTracing` is set to `true`. Do not set `addExtension` to `true` when using `forwarderArn`.                                                                                                                                                                                                                 |
+| `forwarderArn`          | Setting this parameter subscribes the given Datadog forwarder to the Lambda functions’ CloudWatch log groups. Required when `enableDDTracing` is set to `true` unless the subscription is otherwise applied. If a subscription is applied via an integration such as DataDog's AWS Integration, this is not required. Do not set `addExtension` to `true` when using `forwarderArn`.                                                                                                                                                                                                                 |
```

Comment:
Perhaps we should be a bit more specific about the `subscription` and what is `not required`. I think the sentence starting with "If a subscription is ..."
could be rephrased to "For example if a Datadog Forwarder subscription is applied via Datadog's AWS Integration, then `forwarderArn` is not required."